### PR TITLE
Update readme_cn to fully match the main readme

### PR DESCRIPTION
**Pull request by Olvi73**
GitHub Stats Card first note translation
Wakatime Card Exclusive Options translation

### CODE REVIEW

**Comment by Olvi73** [comment]
I commit a full match of chinese version to main readme.md

**Review by Hephaest — APPROVED** [review]
Great job!

File: docs/readme_cn.md [hunk 1]
```diff
@@ -74,7 +75,9 @@
 [![Anurag's github stats](https://github-readme-stats.vercel.app/api?username=anuraghazra)](https://github.com/anuraghazra/github-readme-stats)
 ```
 
-_注: 等级基于用户的统计信息计算得出，详见 [src/calculateRank.js](../src/calculateRank.js)_
+_注: 可用的 ranks 有 S+ (前 1%), S (前 25%), A++ (前 45%), A+ (前 60%), 以及 B+ (每个人)。
+Rank 值是通过你的 commits, contributions, issues, stars, pull requests, followers, 以及拥有的 repositories,
```
Comment (liyishuai) [inline docs/readme_cn.md:79]:
```suggestion
_注：可用的等级有 S+（前 1%）、S（前 25%）、A++（前 45%）、A+（前 60%）以及 B+（其余所有人）。
等级由提交数（commits）、参与项目数（contributions）、提出问题数（issues）、获标星数（stars）、提案数（pull requests）、关注者（followers）、仓库数（repositories）决定，
```

File: docs/readme_cn.md [hunk 2]
```diff
@@ -159,6 +162,8 @@ dark, radical, merko, gruvbox, tokyonight, onedark, cobalt, synthwave, highcontr
 - `include_all_commits` - 统计总提交次数而不是仅统计今年的提交次数 _(boolean)_
 - `count_private` - 统计私人提交 _(boolean)_
 - `line_height` - 设置文本之间的行高 _(number)_
+- `custom_title` - 为卡片设置自定义标题
+- `disable_animations` - 禁用卡片中的所有动画 _(boolean)_
```
Comment (liyishuai) [inline docs/readme_cn.md:166]: 中文夹用英文应使用全角括号，下同。
此遗留问题，可另行修正。

File: docs/readme_cn.md [hunk 3]
```diff
@@ -171,10 +176,22 @@ dark, radical, merko, gruvbox, tokyonight, onedark, cobalt, synthwave, highcontr
...
 - `layout` - 在两个可用布局 `default` & `compact` 间切换
 - `card_width` - 手动设置卡片的宽度 _(number)_
+- `langs_count` - 在卡片上显示更多语言，介于1-10之间，默认为5 _(number)_
+- `exclude_repo` - 排除指定的 repo _(Comma-separated values)_
+- `custom_title` - 为卡片设置自定义标题
 
 > :warning: **重要:**  
 > 如 [Percent Encoding](https://en.wikipedia.org/wiki/Percent-encoding) 所指定，语言名称应使用 uri 转义。  
 > (例: `c++` 应该是 `c%2B%2B`, `jupyter notebook` 应该是 `jupyter%20notebook`, 等.)
+> 你能使用[urlencoder.org](https://www.urlencoder.org/) 来帮助你自动转换。
```
Comment (liyishuai) [inline docs/readme_cn.md:186]:
```suggestion
> 自动转换编码，可使用[urlencoder.org](https://www.urlencoder.org/)。
```

File: docs/readme_cn.md [hunk 4]
```diff
@@ -226,6 +251,14 @@ _注意：热门语言并不表示我的技能水平或类似的水平，它是
 [![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=anuraghazra&hide=javascript,html)](https://github.com/anuraghazra/github-readme-stats)
 ```
 
+### 显示更多语言
+
+你可以使用 `&langs_count=` 选项来增加或减少卡片上显示的语言数量。有效值介于1和10(包含10)之间的整数，默认值为5。
```
Comment (liyishuai) [inline docs/readme_cn.md:256]:
```suggestion
你可以使用 `&langs_count=` 选项来增加或减少卡片上显示的语言数量。有效值为整数从1到10（含），默认值为5。
```

**Review by liyishuai — CHANGES_REQUESTED** [review]
Further localization, including terms and punctuations.

**Pull request closed without merging**